requirements/specifications/media/media-info.md

```diff
@@ -61,263 +58,338 @@ I want to show an audio/videophile overlay with detailed information:
+The `Display` module **MUST** have a `refreshRate` method that returns an number value for the refresh rate that the display supports (in Hz).
 
-### 5.3. Current and Optimal Resolution
+If no display is present, a value of zero is returned.
 
-The `Display` module **MUST** have a `currentResolution` and  `optimalResolution` method that returns a valid resolution for either the current state of the display, or the displays optimal resolution value.
+### 6. Device Properties
 
-This method **MUST** return a value from the `Display.Resolution` enum.
+Apps need to know various aspects of the device, including its media playing capabilities.
 
-The `optimalResolution` **MUST** come from the HDMI edid.
+These will be surfaced in the `Device` module.
 
-For built-in displays `optimalResolution` **MUST** also be provided.
+Access to these APIs is governed by the `xrn:firebolt:capability:device:info` capability.
 
-### 5.4. Supported Resolutions
- 
-The `Display` module **MUST** have a `supportedResolutions` method that returns an array of valid resolutions that the display supports.
+### 6.1. Current Video Mode
 
-This method **MUST** return a value from the `Display.Resolution` enum.
+The `Device` module **MUST** have a `videoMode` method that returns the current video modes used by the device.
 
-### 5.5. Device Supported Resolutions
-The `Device` module **MUST** have a `supportedResolutions` method that returns an array of valid resolutions that the device supports, regardless of any connected display.
+This method **MUST** return a value from the `Media.VideoMode` enum.
 
-This method **MUST** return an array with one or more of the following values from the `Display.Resolution` enum.
+If no display is present, a value of `unknown` is returned.
 
-### 5.6. Use Source Framerate
-The `Display` module **MUST** have a boolean `useSourceFrameRate` API.
+### 6.1. Video Modes Supported
 
-This API **MUST** return `true` if the hdmi output frame rate is set to follow video source frame rate.
+The `Device` module **MUST** have a `videoModesSupported` method that returns an array of valid video modes that the device supports, regardless of any connected display.
 
-Otherwise, this API **MUST** return false.
+This method **MUST** return an array with one or more values from the `Media.VideoMode` enum.
+
+### 6.2 Video Resolution
+
+The `Device` module **MUST** have a `videoResolution` method that returns the width and height of the current video output resolution.
+
+If no display is present, both returned values **MUST** be zero.
+
+### 6.2 Current HDR Profile
+
+The `Device` module **MUST** have a `hdrProfile` method that returns the HDR profile currently used by the device for video output.
+
+This method **MUST** return a value from the `Media.HDRProfile` enum.
+
+### 6.3 Supported HDR Profiles
+
+The `Device` module **MUST** have an `hdrProfilesSupported` method that returns the HDR profiles that the device supports, regardless of any connected display.
```

Comment:
I wonder if we want a different convention for getting all supported things.

`fooSupported` is what we use for detecting if a single value is supported, e.g. `videoFormatSupported`.

`foosSupported` is so easy to mistake for `fooSupported`.

Maybe just `hdrProfiles` or `supportedHdrProfiles`?

Reply:
Thanks for clarifying that; switched to dropping `supported` from the methods that simply return a list of values supported by the platform (e.g. `hdrProfiles`)